### ISSUE TITLE
Bridge Setup: Failed to upgrade database (new sqlite db)

### ISSUE THREAD

**Issue by KeenMaron** [mautrix/signal#178]
Hi!

I followed the setup guide, and now i'm stuck at the last step "python -m mautrix_signal". I created a new Sqlite db before.
After executing "python -m mautrix_signal" i got following error:

![image](https://user-images.githubusercontent.com/52194699/144751800-b0f77978-fd6e-4d76-8e50-cfdf0876a7e8.png)

Any ideas :(? Why the database has to be upgraded?

One wish from my side: Maybe the setup guide could be more detailed. It is really time consuming to collect the missing informations in different disscussions / threads :). (E.g.: It wasn't clear, if i have to use the same database as matrix or have to create a new one; or how to install libolm-dev from backports?)

**Comment by tulir** [comment]
SQLite shouldn't be used, but looks like I forgot to add warnings about it to the example config. The actual problem is probably a missing `/` in the database URI (you only have 2)

**Comment by KeenMaron** [comment]
Ok thanks. Can SQLite cause any problems? I added the third "/" and it worked. Now i'm stuck in another error.
![image](https://user-images.githubusercontent.com/52194699/144754775-96197de5-4666-40b4-887f-0a1735d68537.png)
![image](https://user-images.githubusercontent.com/52194699/144754806-28bd811e-c943-4c84-a574-46b3167b73be.png)

I have disabled registration at my matrix server. Do I have to activate it temporary for this to work?
